pennylane/operation.py

```diff
@@ -709,6 +718,52 @@ def generator(self):
         """
         return [None, 1]
 
+    def parameter_frequencies(self):
+        r"""Returns the frequencies for each operator parameter with respect
+        to an expectation value of the form
+        :math:`\langle \psi | U(\mathbf{p})^\dagger \hat{O} U(\mathbf{p})|\psi\rangle`.
+        These frequencies encode the behaviour of the operator :math:`U(\mathbf{p})`
+        on the value of the expectation value as the parameters are modified.
+        For more details, please see the :mod:`.pennylane.fourier` module.
+
+        Returns:
+            list[tuple[int or float]]: Tuple of frequencies for each parameter.
+                Note that only non-negative frequency values are returned.
+
+        **Example**
+        >>> op = qml.CRot(0.4, 0.1, 0.3, wires=[0, 1])
+        >>> op.parameter_frequencies()
+        [(0.5, 1), (0.5, 1), (0.5, 1)]
+        For operators that define a generator, the parameter frequencies are directly
+        related to the eigenvalues of the generator:
+        >>> op = qml.ControlledPhaseShift(0.1, wires=[0, 1])
+        >>> op.parameter_frequencies()
+        [(1,)]
+        >>> gen_eigvals = tuple(np.linalg.eigvals(op.generator[0] * op.generator[1]))
+        >>> qml.gradients.eigvals_to_frequencies(gen_eigvals)
+        (tensor(1., requires_grad=True),)
```

Comment:
```suggestion
        [(0.5, 1), (0.5, 1), (0.5, 1)]

        For operators that define a generator, the parameter frequencies are directly
        related to the eigenvalues of the generator:

        >>> op = qml.ControlledPhaseShift(0.1, wires=[0, 1])
        >>> op.parameter_frequencies()
        [(1,)]
        >>> gen_eigvals = tuple(np.linalg.eigvals(op.generator[0] * op.generator[1]))
        >>> qml.gradients.eigvals_to_frequencies(gen_eigvals)
        (tensor(1., requires_grad=True),)
```

Line breaks are required for this to render properly :)

https://pennylane--2180.org.readthedocs.build/en/2180/code/api/pennylane.operation.Operation.html#pennylane.operation.Operation.parameter_frequencies

Reply:
Thanks, nice catch!